### PR TITLE
Fix/sort buttons add commas

### PR DESCRIPTION
**Pull request by alexthompson207**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [X] Fix
- [ ] Feature
- [ ] Refactor
- [ ] Style
- [ ] Testing

### Are there any known bugs?

- [X] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Reset sort state in `App` when returning home.
Add function to convert large numbers to include commas.

### What ticket(s) do the changes resolve?
Card about sort bug and card about adding commas to data.

Please Link Issues to this PR as necessary.

### Notes for the Reviewer
Can add the formateLargeNumbers function to any of the data we want!

### How to Test Changes?

Commas should display in Mass, Diameter, and Distance from the sun. When viewing planets details should return home and no planet fact should be displayed.

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -13,6 +13,13 @@ const parseMass = (value: number, exponent: number) => {
   const delta = exponent - 18; //to find exponent beyond 10^18 (quintillion)
   return (value * Math.pow(10, delta));
 }
+
+const formateLargeNumbers = (number: number): string => {
+  let totalFormatted = number.toString().split('.');
+  totalFormatted[0] = totalFormatted[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  return (totalFormatted.join('.'));
+}
```
Comment (ConnorAndersonLarson) [inline src/components/App/App.tsx:21]: Nice! Makes things so much more readable

**Review by ConnorAndersonLarson — APPROVED** [review]
Great work! Thanks for figuring that bug out!

File: src/components/App/App.tsx (hunk 1 above)
Comment (knees4bees) [reply src/components/App/App.tsx:21]: Solid work, Alex!!

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -80,7 +92,7 @@ class App extends React.Component<{}, AllData> {
               return (
                 <>
                   {!foundPlanet && <h2>Oops, looks like that planet is out of our solar system</h2>}
-                  {foundPlanet && <PlanetInfo currentPlanet={foundPlanet} />}
+                  {foundPlanet && <PlanetInfo currentPlanet={foundPlanet} resetSort={this.resetSort} />}
```
Comment (knees4bees) [inline src/components/App/App.tsx:95]: Nice simple fix.

**Review by knees4bees — APPROVED** [review]
Looks great!

**Pull request merged**